grafsy-client/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/leoleovich/grafsy"
+)
+
+var version = "dev"
+
+func main() {
+	var configFile string
+	var connectionTimeout int
+	printVersion := false
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage: %s [args] [file1 [fileN...]]\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "   Or: metrics-generator | %s [args]\n\n", os.Args[0])
+		fmt.Fprintln(os.Stderr, "Reads metrics from files or STDIN and writes to grafsy LocalBind address.")
+		fmt.Fprintf(os.Stderr, "If STDIN contains something, then files will be ignored\n\n")
+		flag.PrintDefaults()
+	}
+	flag.StringVar(&configFile, "c", "/etc/grafsy/grafsy.toml", "Path to config file.")
+	flag.BoolVar(&printVersion, "v", printVersion, "Print version and exit")
+	flag.IntVar(&connectionTimeout, "w", 50, "Timeout ")
+	flag.Parse()
+	args := flag.Args()
+
+	if printVersion {
+		fmt.Printf("Version: %v\n", version)
+		os.Exit(0)
```

Comment:
Maybe instead of all of this magic use loggers with Errorf and Fatalf which exits
logrus is a very good logger.

Reply:
In this particular place, it's the same as in grafsy/main.go, and exit(0) makes perfect sense. For other `exit(1)` I agree but would use the log package (it's already in config.go), and plan to migrate the whole logging to the logrus package.

#21